### PR TITLE
localize quick eval widget

### PR DESCRIPTION
**Pull request by vix633**

### CODE REVIEW

File: activities.serge.json [hunk 1]
```diff
@@ -145,19 +145,22 @@
       "zh-tw zh-tw"
     ]
   },{
-    "name": "assignmentActivityEditor",
```
Comment (jeroach) [inline activities.serge.json:148]: Is this just a rename?

**Review by jeroach — APPROVED** [review]

File: activities.serge.json (hunk 1 above)
Comment (vix633) [reply activities.serge.json:148]: Something changed in master since this i think, let me double check

Comment (vix633) [reply activities.serge.json:148]: Yeah this is weird, i dont know whats with this change.

Comment (jeroach) [reply activities.serge.json:148]: Should the new code be a copy into a new block? Instead of editing an existing block?

Comment (kieranderson) [reply activities.serge.json:148]: Seems like this still got removed?
It's gone in master.
https://github.com/BrightspaceHypermediaComponents/activities/blob/master/activities.serge.json

**Comment by vix633** [comment]
FYI: the diff for this that's showing now, is not what the diff showed at merge time. There's some weird git diff caching weirdness that went down here.

**Pull request merged**